raiden/transfer/mediated_transfer/state_change.py

```diff
@@ -75,6 +75,24 @@ def __post_init__(self) -> None:
         typecheck(self.transfer, LockedTransferSignedState)
 
 
+@dataclass
+class ActionTransferReroute(BalanceProofStateChange):
+    transfer: LockedTransferSignedState
+    secret: Secret = field(repr=False)
+    secrethash: SecretHash = field(default=EMPTY_SECRETHASH)
+
+    def __post_init__(self) -> None:
+        super().__post_init__()
+        typecheck(self.transfer, LockedTransferSignedState)
+
+        self.secrethash = SecretHash(sha256(self.secret).digest())
+
+
+@dataclass
+class ReceiveTransferCancelRoute(BalanceProofStateChange):
```

Comment:
A short docstring explaining when these state changes happen would be nice for both of them.